bmeg_app/views/literature_support_view.py

```diff
@@ -1,17 +1,17 @@
 from .. import appLayout as ly
```

Comment:
Pie chart is dominated by "nulls", we should handle this better otherwise hard to see trends. we could keep all results in our tables but the pie chart filters out null compound taxonomies (and display `N nulls = xx` in the figure title). otherwise defeats the message this figure is to address

![image](https://user-images.githubusercontent.com/40308471/95118307-ee099f80-06fe-11eb-8639-d928c69b3cef.png)